lib/game.rb

```diff
@@ -0,0 +1,86 @@
+require_relative "../lib/players/human.rb"
+require_relative "../lib/players/computer.rb"
+require_relative "../lib/board.rb"
+
+class Game
+  extend Players
+
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 4, 8],
+    [6, 4, 2]
+  ]
+
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    board.turn_count.odd? ? player_2 : player_1
+  end
+
+  def won?
+    combo_x = board.cells.map.with_index { |x, i|
+        i if x == "X"
+    }.compact
+
+    combo_o = board.cells.map.with_index { |o, i|
+        i if o == "O"
+    }.compact
+
+    WIN_COMBINATIONS.find { |combo|
+      (combo_x & combo).size == 3 || (combo_o & combo).size == 3
+    }
+  end
+
+  def draw?
+     !won? && board.full? ? true : false
+  end
+
+  def over?
+    won? || draw? ? true : false
+  end
+
+  def winner
+    won? ? board.position(won?.last + 1) : nil
+  end
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.